docs/from-python.md

```diff
@@ -4,4 +4,90 @@ title: From Python
 sidebar_label: From Python
 ---
 
-Placeholder page
+This guide will help you learn Dark if you're familiar with Python.
+
+Have a tip to add to the guide? [Submit a PR](https://github.com/darklang/docs/pulls)!
+
+## Comments
+
+In Python, you use a `#` to start a comment:
+
+```python
+# This is a comment
+```
+
+In Dark, you use a `let _ =` to start a comment, and you write the comment as a String:
+
+![assets/from-python/comment.png](assets/from-python/comment.png)
+
+## Strings
+
+In Python, you can write a String in multiple ways:
+
+```python
+"This is a string"
+'This is also a string'
+```
+
+In Dark, you always use double quotes:
+![assets/from-python/string.png](assets/from-python/string.png)
+
+## Functions
+
+In Python, you use the `def` keyword to define a function:
+
+```python
```

Comment:
this is a function definition, and dark you used a function call. 

The equivalent  of a function definition in dark is creating a function.